Tests/GRPCHTTP2TransportTests/HTTP2TransportTests.swift

```diff
@@ -199,7 +199,7 @@ final class HTTP2TransportTests: XCTestCase {
       serviceConfig.loadBalancingConfig = [.roundRobin]
       transport = try HTTP2ClientTransport.Posix(
         target: target,
-        config: .defaults {
+        config: .defaults(transportSecurity: .plaintext) {
```

Comment:
Now that we have client and server support for TLS we should add some tests. @simonjbeaumont was going to use `swift-certificates` in NIOSSL to generate test certs in tests rather than futzing around with scripts to regenerate them every year so might be worth syncing with him.

Reply:
We decided we'll do this in a follow-up PR